features/messages/impl/src/main/res/values-sk/translations.xml

```diff
@@ -5,6 +5,11 @@
     <item quantity="few">"%1$d zmeny miestnosti"</item>
     <item quantity="other">"%1$d zmien miestnosti"</item>
   </plurals>
+  <plurals name="screen_room_timeline_more_reactions">
+    <item quantity="one"></item>
```

Comment:
There is a pb here, the value is missing. The CI should detect missing values. :/